### ISSUE TITLE
Detecting screens only in the same USB port

### ISSUE THREAD

**Issue by iansangines** [AdnanHodzic/displaylink-debian#273]
Every time I connect my displaylink dock to my laptop I must do it in one specific USB port. If I connect it in any other USB3.0 port any external screen is detected. 
The `xrandr --list-providers` command only shows integrated screen and the HDMI port provider, but not the dock extra ports.

However, the dock is detected correctly as a DisplayLink device (`lsusb` output).

Any clue about what is happening?

**Comment by medbs** [comment]
Same thing happened to me with a Dell Inspiron 15 7000 Gaming laptop + Debian 9.
I spent almost 2 years dealing with very rare weird behaviors like monitors not being detected, monitors freezing and displaying a frozen screen even after unplugging the docking station,etc.
I also had to always plug the docking station in one particular USB port, probably it has something to do with USB 2, USB 3, USB 3.1 difference. I think that these glitches usually come from the docking station itself, and has nothing to do with the Display Link driver.
